src/parser.c

```diff
@@ -663,7 +663,8 @@ char *parser_data_create_object(const char *device_id, uint8_t sensor_id,
 		puts("The localtime() function failed");
 		has_err = true;
 	}
-	strftime(buf, 32, "%Y-%m-%dT%X.0%z", ptm);
+	else
+		strftime(buf, 32, "%Y-%m-%dT%X.0%z", ptm);
```

Comment:
Put theses braces to keep the coding style
(https://www.kernel.org/doc/html/v4.10/process/coding-style.html#placing-braces-and-spaces)

```suggestion
	} else {
		strftime(buf, 32, "%Y-%m-%dT%X.0%z", ptm);
	}
```